invesalius/gui/task_navigator.py

```diff
@@ -737,19 +738,23 @@ def __init__(self, parent):
             label = fiducial['label']
             tip = fiducial['tip']
 
-            self.btns_set_fiducial[n] = wx.ToggleButton(self, button_id, label=label, size=wx.Size(45, 23))
-            self.btns_set_fiducial[n].SetToolTip(wx.ToolTip(tip))
-            self.btns_set_fiducial[n].Bind(wx.EVT_TOGGLEBUTTON, partial(self.OnImageFiducials, n))
+            ctrl = wx.ToggleButton(self, button_id, label=label, size=wx.Size(45, 23))
```

Comment:
If you remove the size it shows wrong?

Reply:
@tfmoraes If I remove the size parameter, the image fiducial buttons look like this:

![image](https://user-images.githubusercontent.com/49641103/132205440-44a2bd27-1c4e-4c34-810a-000134e4d710.png)

That is, the button takes the whole horizontal space instead of just enough space to fit in "LEI" etc. text. (Compare to the "LET" etc. buttons, which still have the size parameter.)

Could you elaborate on what is the change that you propose -- for instance, would you prefer the buttons to not have the size parameter? Thanks!

Reply:
You can import and use [ `calc_width_needed ` ](https://github.com/invesalius/invesalius3/blob/eddf4890375de4c3773adc931cab3b1ec179f14f/invesalius/gui/utils.py#L23) to calculate the width need to keep the text, something like this:

```
ctrl = wx.ToggleButton(self, button_id, label=label)
ctrl.SetMinSize((calc_width_needed(ctrl, 3), -1))
```

`3` is the number of characters. `-1` is because you want to the default value to height.

Reply:
Thanks, done.